app/models/spree/gateway/payu.rb

```diff
@@ -14,6 +14,7 @@ class Gateway::Payu < PaymentMethod
     preference :test_mode, :boolean, default: false
     preference :return_url, :string, default: "http://localhost:3000"
     preference :return_status_url, :string, default: "http://localhost:3000"
+    preference :delivery_limit_czk, :integer
```

Comment:
czy te pole nie będzie teraz widoczne dla każdej metody płatności? Myślałeś nad zrobieniem z tego pola, dzięki któremu będzie można ustalać limity dla jakiejkolwiek metody płatności?
```suggestion
    preference :max_payment_amount, :integer
    preference :min_payment_amount, :integer
```
I nastepnie serwis typu CheckPaymentLimits(order), w ktorym będzie publiczny interfejs 
def call
  case name
  when 'Twisto'
    order.total < twisto_max && order.total > twisto_min
  when 'AnyOtherPayment'
    ...
  ...
  else
    true
  end
  
alboo jeszcze lepiej jakby udalo się duck typing tu zastosować. W kazdym razie dobrze by było zrobić z tego coś bardziej uniwersalnego

Reply:
W tej chwili zadanie ma tylko maksymalny limit i jedną metodę płatności. Jeśli jest tego więcej, możemy do tego dodać serwis. Jeszcze nie widzę sensu

Reply:
w wymaganiach dla płatności na veronie są zarówno dolne jak i górne limity już ustalone nie tylko dla twisto, ale też dla payu rat i blika np. A myślę, że tam również będziemy korzystać z tego gema
https://docs.google.com/spreadsheets/d/1RNbW5V08CKphVI_ZlP6AGPNL_MduDIef9Gpaa--jxiU/edit?gid=0#gid=0